packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/delete/delete.routing.js

```diff
@@ -59,6 +60,7 @@ export default /* @ngInject */ ($stateProvider) => {
           serviceId: service.domain,
           serviceType: service.serviceType,
           renew: service.renew,
+          route: get(service, 'route.url'),
```

Comment:
```suggestion
          route: service.route?.url
```
not possible ? (that way we avoid a `_.get`